lib/mjml/parser.rb

```diff
@@ -15,25 +15,27 @@ def initialize
     end
 
     def call(template)
-      exec!(template)
+      @template = template
```

Comment:
It was more convenient as I use it in a lot of method. I'm avoiding to pass the template each time ...